models/demos/tg/llama3_70b/tests/test_llama_model_galaxy.py

```diff
@@ -143,36 +145,43 @@ def run_test_LlamaModel_inference(
 
         start_pos = generation_start_pos + i
 
-        # PyTorch output --------------------------------------------------------------------
+        # # PyTorch output --------------------------------------------------------------------
+        # logger.info(f"Running inference on PyTorch")
```

Comment:
I will uncomment, it's super useful to track what stage is running currently especially when longer seq_lens since everything runs longer and might look like a hang while it's just long.